src/app/settings/organizations.component.ts

```diff
@@ -88,32 +104,56 @@ export class OrganizationsComponent implements OnInit {
     }
 
     async toggleResetPasswordEnrollment(org: Organization) {
-        // Feature Flag
-        if (!this.resetPasswordFeatureFlag) {
-            return;
-        }
-
         // Set variables
         let keyString: string = null;
         let toastStringRef = 'withdrawPasswordResetSuccess';
 
-        // Enroll - encrpyt user's encKey.key with organization key
+        // Enrolling
         if (!org.resetPasswordEnrolled) {
-            const encKey = await this.cryptoService.getEncKey();
-            const orgSymKey = await this.cryptoService.getOrgKey(org.id);
-            const encryptedKey = await this.cryptoService.encrypt(encKey.key, orgSymKey);
-            keyString = encryptedKey.encryptedString;
-            toastStringRef = 'enrollPasswordResetSuccess';
-        }
-
-        // Create/Execute request
-        try {
+            // Alert user about enrollment
+            const confirmed = await this.platformUtilsService.showDialog(
+                this.i18nService.t('resetPasswordEnrollmentWarning'), null,
+                this.i18nService.t('yes'), this.i18nService.t('no'), 'warning');
+            if (!confirmed) {
+                return;
+            }
+
+            // Retrieve Public Key
+            this.actionPromise = this.apiService.getOrganizationKeys(org.id)
+                .then(async response => {
+                    let publicKey = null;
+
+                    if (response != null) {
+                        publicKey = Utils.fromB64ToArray(response.publicKey);
+                    } else {
+                        throw new Error('Get Organization Keys response is null');
```

Comment:
Do need to localize error messages; although this one doesn't really mean anything to anyone other than perhaps Customer Success.

Reply:
Updated.